bin/lib/check_reqs.js

```diff
@@ -362,6 +362,36 @@ module.exports.run = function () {
     });
 };
 
+/** Checks if Windows SDK required to build the target_platform is present
+ * @param {String}  target_platorm        Target platform ('8.1' or '10.0')
+ * @return {Boolean}                      Whether the min SDK required to build the platform is present  
+ *
+ */ 
+module.exports.isWinSDKPresent = function (target_platform) {
+    return getInstalledWindowsSdks()
+    .then(function (installedSdks) {
+        var reqVersion = Version.tryParse(REQUIRED_VERSIONS[target_platform].windowssdk);
+        var hasSdkInstalled = installedSdks.some(function (installedSdk) {
+            return installedSdk.eq(reqVersion);
+        });
+
+        return hasSdkInstalled ? true : false;
```

Comment:
This line is excess. `return installedSdks.some(function (installedSdk) {...` should be sufficient